extension/src/popup/components/sendPayment/SendSettings/Settings/index.tsx

```diff
@@ -133,11 +133,13 @@ export const Settings = ({
             ),
             CLASSIC_ASSET_DECIMALS,
           );
-          dispatch(
-            saveTransactionFee(
-              baseFee.plus(new BigNumber(minResourceFee)).toString(),
-            ),
-          );
+          if (!transactionFee) {
```

Comment:
we do a similar check in the other pathways - this keeps the effect from running indefinitely